<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" idbasedtr="false">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">this-&gt;setStyleSheet(&quot;background-color: white;&quot;);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">this-&gt;setStyleSheet(&quot;background-color: white;&quot;);</string>
   </property>
   <widget class="QTabWidget" name="tab">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>731</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab { height: 35px; width: 180px
; }</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::South</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <widget class="QWidget" name="tab_LiveData">
     <attribute name="title">
      <string>Live Daten</string>
     </attribute>
     <widget class="QFrame" name="frame_topleft">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>330</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame#frame_topleft {background-color: white};</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <widget class="QLabel" name="lab_DataType_topleft">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lab_Data_topleft">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>331</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>40</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_topright">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>5</y>
        <width>330</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame#frame_topright {background-color: white};</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <widget class="QLabel" name="lab_DataType_topright">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lab_Data_topright">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>331</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>40</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_bottomright">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>175</y>
        <width>330</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame#frame_bottomright {background-color: white};</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <widget class="QLabel" name="lab_DataType_bottomright">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lab_Data_bottomright">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>331</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>40</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_bottomleft">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>175</y>
        <width>330</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame#frame_bottomleft {background-color: white};</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <widget class="QLabel" name="lab_DataType_bottomleft">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lab_Data_bottomleft">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>331</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>40</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>721</width>
        <height>371</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="lab_UserTitle">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>330</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Username: </string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_BoatTitle">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>330</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Bootsname:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_Boat">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>330</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lab_User">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>330</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <zorder>frame_5</zorder>
     <zorder>frame_bottomleft</zorder>
     <zorder>frame_bottomright</zorder>
     <zorder>frame_topright</zorder>
     <zorder>frame_topleft</zorder>
    </widget>
    <widget class="QWidget" name="tab_LiveDataGraphs">
     <attribute name="title">
      <string>Live Graphen</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>641</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Settings">
     <attribute name="title">
      <string>Einstellungen</string>
     </attribute>
     <widget class="QGroupBox" name="GB_BottomRight">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>300</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Messwert - Unten rechts</string>
      </property>
      <widget class="QComboBox" name="comboBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>22</y>
         <width>281</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="GB_BottomLeft">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>300</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Messwert - Unten links</string>
      </property>
      <widget class="QComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>22</y>
         <width>281</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="GB_TopRight">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>300</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Messwert - Oben rechts</string>
      </property>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>22</y>
         <width>281</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="GB_TopLeft">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>35</y>
        <width>300</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Messwert - Oben links</string>
      </property>
      <widget class="QComboBox" name="comboBox_1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>22</y>
         <width>281</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="GB_TopLeft_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>30</y>
        <width>321</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Benutzerauswahl</string>
      </property>
      <widget class="QComboBox" name="comboBox_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>21</y>
         <width>301</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="GB_TopLeft_3">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>110</y>
        <width>321</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Bootsauswahl</string>
      </property>
      <widget class="QComboBox" name="comboBox_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>21</y>
         <width>301</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pB_AddUser">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>260</y>
        <width>75</width>
        <height>75</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pB_AddBoat">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>260</y>
        <width>75</width>
        <height>75</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>230</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nutzer hinzufügen</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>230</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Boot hinzufügen</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pB_AddUser_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>180</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Vorlage speichern</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_StartStop">
     <attribute name="title">
      <string>Start / Stop</string>
     </attribute>
     <widget class="QPushButton" name="pB_StartTraining">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Training starten</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pB_StopTraining">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>150</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Training stoppen</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pB_Off">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>260</y>
        <width>251</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Schlagzahluhr ausschalten</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>5</y>
      <width>16</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lab_Rowtech">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Modern No. 20</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RowTech</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_Datum">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>90</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>28.12.1995</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>5</y>
      <width>16</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lab_Zeit">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>90</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>19:45:25</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>5</y>
      <width>16</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lab_GPS">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>5</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GPS Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_MPU">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>16</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MPU6050 Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_GPS_stat">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>5</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_MPU_stat">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>16</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_GPS_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>25</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GPS Qualität:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_GPS_qual">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>25</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
